### PR TITLE
doc: added documentation for new features in 1.4 release

### LINKED ISSUE #139: doc: Documentation 1.4 release
Update corresponding documentation sections:

- [x] Retail customers
- [x] Roadwarrior users
- [x] Record locutions service
- [x] Listen to voicemail from any terminal
- [x] Import calendar dates
- [x] Changes in brand SIP domains
- [x] Video support
- [ ] Changes in invoices

### PR DESCRIPTION
**Pull request by Kaian**
* Listen to voicemail from any terminal
  * Changes in brand SIP domains
  * Record locutions service

Fixes #139

### CODE REVIEW

File: doc/sphinx/locale/es/LC_MESSAGES/pbx_features.po [hunk 1]
```diff
@@ -2157,13 +2156,30 @@ msgstr "Consultar el buzón de voz"
 msgid ""
 "This service allows checking the user's voicemail using an incteractive "
```
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/pbx_features.po]: incteractive => interactive

File: doc/sphinx/locale/es/LC_MESSAGES/pbx_features.po [hunk 2]
```diff
@@ -2157,13 +2156,30 @@ msgstr "Consultar el buzón de voz"
...
+"service allows optional extension after the service code to check other "
+"users voicemails. Users can protect their voicemail using the internal "
+"menu options."
 msgstr ""
 "Este servicio permite acceder a un menú de voz que te presenta los "
-"mensajes de voz nuevos, viejos, etc. Es una alternativa a la recepción de"
-" mensajes de voz vía correo electrónico."
+" mensajes de voz nuevos, viejos, etc. Es una alternativa a la recepción "
+"de mensajes de voz vía correo electrónico. A partir de 1.4, este servicio "
+"acepta una extensión opcional trás el código de sercicio para poder acceder "
```
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/pbx_features.po]: sercicio => servicio

File: doc/sphinx/pbx_features/services.rst [hunk 3]
```diff
@@ -7,76 +7,93 @@ Additional services
...
+
+.. ifconfig:: language == 'es'
+
+    .. image:: img/es/services_god.png
 
 .. glossary::
 
     Direct pickup
-        This service allows capturing a ringing call from other terminal by 
+        This service allows capturing a ringing call from other terminal by
```
Comment (mmadariaga) [inline doc/sphinx/pbx_features/services.rst]: another?

File: doc/sphinx/pbx_features/services.rst [hunk 4]
```diff
@@ -7,76 +7,93 @@ Additional services
...
     Group pickup
-        This service allows capturing a ringing call for any terminal whose user 
+        This service allows capturing a ringing call for any terminal whose user
         is part of one of the capturer pickup groups.
 
     Check voicemail
-        This service allows checking the user's voicemail using an incteractive 
-        menu from which new voicemails can be listen, deleted, etc. This an 
-        active alternative to receive voicemails via the email.
+        This service allows checking the user's voicemail using an incteractive
```
Comment (mmadariaga) [inline doc/sphinx/pbx_features/services.rst]: interactive

File: doc/sphinx/pbx_features/services.rst [hunk 5]
```diff
@@ -7,76 +7,93 @@ Additional services
...
+        menu from which new voicemails can be listen, deleted, etc. This an
```
Comment (mmadariaga) [inline doc/sphinx/pbx_features/services.rst]: This ¿is?

**Comment by Kaian** [comment]
Thanks for the review, added changes as suggested.

**Comment by Kaian** [comment]
I have also added the initial version of changelog file for 1.4 release.

**Comment by mmadariaga** [comment]
Approved :smile:

File: doc/sphinx/locale/es/LC_MESSAGES/security.po [hunk 6]
```diff
@@ -79,7 +82,8 @@ msgstr "Puerto 5061 (TCP)"
 msgid ""
 "Port 7060 (TCP/UDP) y 7061 TCP (just in case both ProxyUsers and "
 "ProxyTrunks share IP)"
-msgstr "Puerto 7060 (TCP/UDP) y 7061 TCP (para casos en que ambos "
+msgstr ""
+"Puerto 7060 (TCP/UDP) y 7061 TCP (para casos en que ambos "
 "proxiescompartan la misma dirección IP)"
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/security.po]: _proxiescompartan_

File: doc/sphinx/locale/es/LC_MESSAGES/security.po [hunk 7]
```diff
@@ -129,102 +135,218 @@ msgstr "Se puede activar en la sección **Configuración de Marca** > **Empresas
...
+"their companies are still protected with IP filter mechanism."
+msgstr ""
+"Para solventar este problema, en la sección de Usuarios existe la opción "
+"**Llamadas desde IPs no autorizadas** que permite que ciertos usuarios "
+"llamen desde redes no permitidas mientras su empresa sigue protegida con "
+"el mecanismo de protección de IP. "
+
+#: ../../security/index.rst:85
+msgid ""
+"When users like theese call from non-granted IPs, their amount of "
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/security.po]: _theeeese_

File: doc/sphinx/locale/es/LC_MESSAGES/security.po [hunk 8]
```diff
@@ -129,102 +135,218 @@ msgstr "Se puede activar en la sección **Configuración de Marca** > **Empresas
...
+"are counted and limited, received calls are not affected by this setting "
+"(they are controlled with **MaxCalls** setting)."
+msgstr ""
+"Solo se cuentan y se limitan las llamadas generadas por estos usuarios "
+"(tanto las internas como las externas), la recepción de llamadas no se ve "
+"afectada por esta funcionalidad (sí por el parámetro **Límite de llamadas**). "
+
+#: ../../security/index.rst:92
+msgid "To sum up, with this feature:"
+msgstr "En resumen, con esta funcionalidad: "
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/security.po]: Extra space not harmful yet not required.

File: doc/sphinx/security/index.rst [hunk 9]
```diff
@@ -72,6 +72,49 @@ Both IP addresses or ranges can be used, in CIDR format (IP/mask):
...
+-----------------
+
+Some companies have roadwarrior users that travel often and connect from external
+networks, forcing Companies to disable the IP filter security mechanism.
+
+To solve this issue, there is a user option called **Calls for non-granted IPs**
+that enables these users to call from non-granted IPs while their companies are
+still protected with IP filter mechanism.
+
+When users like theese call from non-granted IPs, their amount of concurrent
```
Comment (Kaian) [inline doc/sphinx/security/index.rst]: Previous mentioned _theese_

**Comment by Kaian** [comment]
I'll wait for @mmadariaga to finish the pending points in #139 before merging :tada:

**Pull request merged**